internal/processor/printer.go

```diff
@@ -105,6 +106,12 @@ func (p *SDCPrinter) ReProcess(pfr *pkg.PrintFileRequest) error {
 	// increment the number of attempts
 	numberOfAttempts := pfr.Attempts
 	pfr.Attempts = numberOfAttempts + 1
+	// log an error for each retry
+	if pfr.Attempts > 1 {
+		strAttempts := strconv.Itoa(pfr.Attempts)
+		retryFile := pfr.DataFilename
```

Comment:
DataFilename is the name of the file received from action and not the print file.
ie.
```
retryFile := pfr.PrintFilename
```